### DISCUSSION TITLE
Add Download button in search result

### DISCUSSION THREAD

**Discussion by wooriele** [filebrowser/filebrowser#1608] [Ideas]
Hi,
Currently, after searched, we can only view and modify the content of file searched. How we can add download button next to the file name in the search result?
Thanks

**Comment by jniggemann** [comment]
I think this is a good idea and should be a feature request.

**Reply by wooriele** [reply]
so I need to create new a feature request or admin/mod can move this thead to Feature Request?

**Reply by jniggemann** [reply]
That's up to the project owner to decide, I commented as a user.
